lib/Dialect/SPIRV/SPIRVOps.cpp

```diff
@@ -750,11 +750,12 @@ struct ConvertChainedBitcast : public OpRewritePattern<spirv::BitcastOp> {
     }
 
     rewriter.replaceOpWithNewOp<spirv::BitcastOp>(
-        bitcastOp, bitcastOp.result()->getType(), parentBitcastOp.operand());
+        {parentBitcastOp.result()}, bitcastOp, bitcastOp.result()->getType(),
```

Comment:
Nit: ```/*valuesToRemoveIfDead=*/{parentBitcastOp.result()}, ```
for better readability.

Reply:
@bondhugula thanks for looking on this! Updated.